### ISSUE TITLE
Sharing from desktop doesn't inherently offer groups, only users

### ISSUE THREAD

**Issue by MiragePV** [owncloud/client#4362]
I am using Win 10, client ver. 2.1.1. nightly, build 5757.

When I am trying to share a folder with some users or groups from the desktop, the following happens:
- I right-click on a folder, select Share with OwnCloud
- A dialog opens, however it doesn't pop out, it just flashes on toolbar (which shouldn't happen, imo, but this is not the issue I want to address)
- I click into the box, and write a whitespace
- A dropdown menu with all possible users pops, but it contains only users, and not groups.
- However if I start writing the name of a group, it suddenly appears in the dropdown menu as "foobar (group)".

The question is, why do groups not show in the dropdown together with the users? And why is there a necessity to write a blank space to show all possible users to share with? Why isn't the box automatically a dropdown menu which can we write into? In the web interface I need to write not one, but two spaces to show a dropdown menu. Why is that? It seems very counterintuitive to me, and a bit impractical.

**Comment by Dianafg76** [comment]
I tested this issue, and in my Windows 10 is working OK
1. I have 1 group (e.g Hello), and 1 user (e.g Hola)
2. I created a new folder and I have shared with users and groups
3. If I writing the letter H, I can see the user Hola and the group Hello 
Desktop v 2.1.1 (build 5765)
Server v 8.2.2

**Comment by ogoffart** [comment]
> A dialog opens, however it doesn't pop out, it just flashes on toolbar

We tried to fix this problem already, but it's not as simple as it seems the window manager prevent us to popup arbitrary window.

As for what shows up on the popup, it's simply what the server answers us for our search query.  So it's most likely just a server problem.
What's the server version?

**Comment by MiragePV** [comment]
@Dianafg76 Yes, that's what I said. When you star typing the group name, it shows.
But if you don't remember all the group names, and you want them all shown, it's impossible to do so. It shows only users.

@ogoffart Server version is 8.2.2 (stable)

**Comment by rullzer** [comment]
As @ogoffart suggested it is likely a way in wich the backend you use searches. If you properly setup your users and have `firstname lastname` then searching for ` ` will match all users since they all have a space in their name.

Now your groups most likely do not have a space. So nothing is found.
